lib/WebpackOptionsDefaulter.js

```diff
@@ -42,7 +42,8 @@ function WebpackOptionsDefaulter() {
 	this.set("node.console", false);
 	this.set("node.process", true);
 	this.set("node.global", true);
-	this.set("node.buffer", true);
+	this.set("node.Buffer", true);
+	this.set("node.setImmediate", true);
```

Comment:
I was wondering about that too, but`setImmediate` and `clearImmediate` are imho closely linked together. You wouldn't want to specify individual options for that. If there's a `setImmediate` it needs a `clearImmediate` as well. And since you always think of `setImmediate` and barely need `clearImmediate` I decided to call it `setImmediate`.